lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -111,7 +111,7 @@ def define
         end
 
         next_node do
-          if %w(italy sweden).include?(calculator.ceremony_country)
+          if %w(italy sweden mozambique).include?(calculator.ceremony_country)
```

Comment:
`Send all outcomes for Mozambique to the generic countries outcome` does not read very well. I don't think we are sending all outcomes to another outcome. I would say we are using the outcome for flattened content in Mozambique.

I would rewrite the commit title and notes accordingly.